doc/nrf/libraries/caf/buttons_pm_keep_alive.rst

```diff
@@ -0,0 +1,26 @@
+.. _caf_buttons_pm_keep_alive:
+
+CAF: Power manager keep alive module for buttons
```

Comment:
typo in commit message: `any configured button is pressed`

Reply:
Ok